igcollect/rsyslog_impstats.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+"""igcollect - Rsyslog impstats parser
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+import re
+from argparse import ArgumentParser
+from time import time
+
+def parse_log():
+    actions = {}
+    args = parse_args()
```

Comment:
No additional args parsing inside helper methods. Pass all needed variables as parameters to the method.

Reply:
Fixed in https://github.com/innogames/igcollect/pull/128/commits/f6e2541aba498c489c3c013379edf6b909546b03